cms/envs/help_tokens.ini

```diff
@@ -12,9 +12,9 @@ updates = course_author:course_assets/handouts_updates.html
 pages = course_author:course_assets/pages.html
 files = course_author:course_assets/course_files.html
 textbooks = course_author:course_assets/textbooks.html
-schedule = course_author:set_up_course/studio_add_course_information/index.html
+schedule = course_author:set_up_course/creating_course/index.html
```

Comment:
The original PR doesn't change help tokens.  Are you sure this one should?

Reply:
Yes, it's true that the original PR didn't change tokens. I changed them because e.g.:

http://edx.readthedocs.io/projects/open-edx-building-and-running-a-course/en/open-release-ginkgo.master/set_up_course/studio_add_course_information/index.html

gives `Permission Denied`

whereas:
http://edx.readthedocs.io/projects/open-edx-building-and-running-a-course/en/open-release-ginkgo.master/set_up_course/creating_course/index.html.

exists.
Not sure why this wasn't caught in any of the previous PRs.

Reply:
We've had these issues before, and now I'm forgetting the details.  Your change looks right, so we'll keep it. Thanks.